src/agent/src/rpc.rs

```diff
@@ -1040,6 +1040,15 @@ impl protocols::agent_ttrpc::AgentService for AgentService {
         // destroy all containers, clean up, notify agent to exit
         // etc.
         sandbox.destroy().await.unwrap();
+        // Close get_oom_event connection,
+        // otherwise it will block the shutdown of ttrpc.
+        let _ = sandbox
```

Comment:
why not just drop this channel to notify peer exit?

Reply:
because the tx is also hold by others

Reply:
When you do the destroy sandbox, all of the containers had exited and the run_oom_event_monitor had exited, thus there's no one take the tx except the sandbox itself, thus you can just drop it thus all of the tx copy would dropped.

Reply:
done